doc/changes/latest.inc

```diff
@@ -188,3 +188,5 @@ API changes
 - Split :func:`mne.viz.Brain.show_view` argument ``view`` into ``azimuth``, ``elevation`` and ``focalpoint`` for clearer view setting and make the default for ``row`` and ``col`` apply to all rows and columns (:gh:`9596` by `Alex Rockhill`_)
 
 - Deprecate ``solver='prox'`` in :func:`mne.inverse_sparse.mixed_norm` in favor of ``solver='cd'`` and ``solver='bcd'`` as coordinate descent solvers consistently outperform proximal gradient descent (:gh:`9608` by `Pierre-Antoine Bannier`_)
+
+- All `mne.connectivity` functions are deprecated to be fully removed in v0.25 (:gh:`9493` by `Adam Li`_).
```

Comment:
Might be good to mention that they got moved - now it sounds as they just got deleted completely.

Reply:
Added mention to mne-conn.